snowflake/repository.go

```diff
@@ -115,10 +115,10 @@ func (repo *SnowflakeRepository) GetDataUsage(ctx context.Context, minTime time.
 
 		query := fmt.Sprintf(`SELECT QUERY_HISTORY.QUERY_ID as QUERY_ID, QUERY_HISTORY.QUERY_TEXT as QUERY_TEXT, DATABASE_NAME, SCHEMA_NAME, QUERY_TYPE, SESSION_ID, QUERY_HISTORY.USER_NAME as USER_NAME, ROLE_NAME, EXECUTION_STATUS, START_TIME, END_TIME, TOTAL_ELAPSED_TIME, BYTES_SCANNED, BYTES_WRITTEN, BYTES_WRITTEN_TO_RESULT, ROWS_PRODUCED, ROWS_INSERTED, ROWS_UPDATED, ROWS_DELETED, ROWS_UNLOADED, CREDITS_USED_CLOUD_SERVICES, DIRECT_OBJECTS_ACCESSED, BASE_OBJECTS_ACCESSED, OBJECTS_MODIFIED, OBJECT_MODIFIED_BY_DDL, PARENT_QUERY_ID, ROOT_QUERY_ID
 FROM "SNOWFLAKE"."ACCOUNT_USAGE"."QUERY_HISTORY" INNER JOIN "SNOWFLAKE"."ACCOUNT_USAGE"."ACCESS_HISTORY" ON QUERY_HISTORY.QUERY_ID = ACCESS_HISTORY.QUERY_ID
-WHERE START_TIME >= ? %s
-ORDER BY QUERY_HISTORY.START_TIME DESC`, endTime)
+WHERE START_TIME >= ? %s`, endTime)
 
 		i := 0
+		mostRecentQueryId := ""
```

Comment:
wondering if it would make more sense to do this on START_TIME instead? QueryId is a string if I remember correctly? Would be inefficient to sort on + it's not the real query order?

Reply:
Thought it was a time-based UUID.
And reason for using it in the first place was to be sure we don't loose items that are just started with the same start_time.